test/component/Legend.spec.tsx

```diff
@@ -1339,7 +1426,6 @@ describe('<Legend />', () => {
         align: LegendProps['align'];
         verticalAlign: LegendProps['verticalAlign'];
         layout: LegendProps['layout'];
-        expectedStyle: string;
```

Comment:
As a bonus - the Legend now renders with the final styles, right away. While before it required render, then read from DOM, and adjustment on second render.